code/modules/mob/living/basic/syndicate/syndicate.dm

```diff
@@ -24,56 +15,55 @@
 	icon_gib = "syndicate_gib"
 	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
 	sentience_type = SENTIENCE_HUMANOID
-	speak_chance = 0
-	turns_per_move = 5
-	speed = 0
-	stat_attack = HARD_CRIT
-	robust_searching = 1
 	maxHealth = 100
 	health = 100
-	harm_intent_damage = 5
+	basic_mob_flags = DEL_ON_DEATH
+	speed = 1.1
+	environment_smash = ENVIRONMENT_SMASH_STRUCTURES
 	melee_damage_lower = 10
 	melee_damage_upper = 10
 	attack_verb_continuous = "punches"
 	attack_verb_simple = "punch"
 	attack_sound = 'sound/weapons/punch1.ogg'
 	combat_mode = TRUE
-	loot = list(/obj/effect/mob_spawn/corpse/human/syndicatesoldier)
-	atmos_requirements = list("min_oxy" = 5, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
+	habitable_atmos = list("min_oxy" = 5, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0)
 	unsuitable_atmos_damage = 7.5
+	unsuitable_cold_damage = 7.5
+	unsuitable_heat_damage = 7.5
 	faction = list(ROLE_SYNDICATE)
-	check_friendly_fire = 1
-	status_flags = CANPUSH
-	del_on_death = 1
-	dodging = TRUE
-	rapid_melee = 2
-	footstep_type = FOOTSTEP_MOB_SHOE
+	ai_controller = /datum/ai_controller/basic_controller/syndicate
+	
+	var/loot = list(/obj/effect/mob_spawn/corpse/human/syndicatesoldier)
 
-///////////////Melee////////////
+/mob/living/basic/syndicate/Initialize(mapload)
+	. = ..()
+	if(LAZYLEN(loot))
+		AddElement(/datum/element/death_drops, loot)
+	AddElement(/datum/element/footstep, footstep_type = FOOTSTEP_MOB_SHOE)
 
-/mob/living/simple_animal/hostile/syndicate/space
+/mob/living/basic/syndicate/space
 	icon_state = "syndicate_space"
 	icon_living = "syndicate_space"
 	name = "Syndicate Commando"
 	maxHealth = 170
 	health = 170
-	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
-	minbodytemp = 0
-	speed = 1
+	loot = list(/obj/effect/gibspawner/human)
+	habitable_atmos = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
```

Comment:
```suggestion
	unsuitable_atmos_damage = 0
```
doing this makes them invulnerable to atmos as well and also prevents them from getting the atmos_requirements element since they wouldn't need it in that case

Reply:
sorry i forgot to mention you should probably do it for the other space variations as well so that it's consistent

Reply:
guh